lib/DDG/Spice/SalesTaxZipCode.pm

```diff
@@ -0,0 +1,20 @@
+package DDG::Spice::SalesTaxZipCode;
+#ABSTRACT: Returns the sales tax for any US 5 digit zipcode. If query has trigger words and 5 digit zipcode then it will process.
+
+use strict;
+use DDG::Spice;
+use Locale::SubCountry;
+
+triggers any => 'sales tax for', 'sales tax in', 'sales tax';
```

Comment:
The first two should be `start` triggers, the last should be a `startend` trigger